charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses/eks-cluster-class.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: eks
+  name: eks-quick-start
+spec:
+  controlPlane:
+    ref:
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta2
+      kind: AWSManagedControlPlaneTemplate
+      name: eks-quick-start-control-plane
+  infrastructure:
+    ref:
+      apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+      kind: AWSManagedClusterTemplate
+      name: eks-quick-start
+  patches:
+  - external:
+      discoverVariablesExtension: eksclusterconfigvars-dv.cluster-api-runtime-extensions-nutanix
+      generateExtension: eksclusterv4configpatch-gp.cluster-api-runtime-extensions-nutanix
+    name: cluster-config
+  - external:
+      discoverVariablesExtension: eksworkerconfigvars-dv.cluster-api-runtime-extensions-nutanix
+      generateExtension: eksworkerv4configpatch-gp.cluster-api-runtime-extensions-nutanix
+    name: worker-config
+  - definitions:
+    - jsonPatches:
+      - op: add
+        path: /spec/template/spec/identityRef
+        value:
+          kind: AWSClusterControllerIdentity
```

Comment:
is this required? Wont it default to CAPA controller if no identity is provided?
Also I this will be replaced using CAREN API in following PRs?

Reply:
I see this in the AWS example too, probably where it came from in the original PR.
But yes this will be replaced by a proper API.